third_party/blink/renderer/core/css/css_numeric_literal_value.cc

```diff
@@ -255,6 +255,15 @@ String CSSNumericLiteralValue::CustomCSSText() const {
           text = FormatInfinityOrNaN(value, UnitTypeToString(GetType()));
         } else {
           text = FormatNumber(value, UnitTypeToString(GetType()));
+
+          // [RUN-1918] Workaround divergent floating point sprintf.
+          std::string textStr = text.Ascii();
+          size_t recordedLength = recordreplay::RecordReplayValue(
+              "CSSNumericLiteralValue::CustomCSSText", textStr.length());
+          recordreplay::RecordReplayBytes(
+              "CSSNumericLiteralValue::CustomCSSText", &textStr[0],
```

Comment:
During replay, we're just writing to textStr as a dumb buffer; how do we know it has enough space for us?

Reply:
Whoops! You are absolutely right. I forgot the string resizing because I originally tried a different approach. Fixed now 😮‍💨